### DISCUSSION TITLE
Manually adding an url to a container

### DISCUSSION THREAD

**Discussion by KrisCvetkovikj** [mozilla/multi-account-containers#2694] [Ideas]
Hi. 
Before all, I like to mention that I've been using this extension for a long time, probably since release, and I thank you for it.
What I'd like to request your audience for is to discuss on a possible feature for the extension. Mind you, I use it alongside Temporary Containers.
Often I come across redirect urls which I've been unable to add to a container mainly because they have no content and if I stop the page load manually with the escape button, it just reads like an about:blank page for the extension. 
This leads to excessive tab opening and closing due to converting from a tmp and regular container. I have the "Limit to designated sites" on since I don't want to have web data across pages that shouldn't share that particular data i.e. I just hate intrusive analytics.

I've seen how the extension keeps the data and I am thinking this is doable but of course, it's just a simple opinion based on a shallow research.

**Comment by influential-eliot** [comment]
This has been raised so many times in the issues, and is a vital request alongside ACTUAL organisation of position in the list.

Thank you, @KrisCvetkovikj ! 👏
